Tests/Unit/Model/SepaDirectDebitPaymentMethodTest.php

```diff
@@ -0,0 +1,112 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+use Wirecard\Oxid\Model\SepaCreditTransferPaymentMethod;
+use Wirecard\Oxid\Model\SepaDirectDebitPaymentMethod;
+use Wirecard\Oxid\Model\Transaction;
+
+use OxidEsales\Eshop\Core\Field;
+use OxidEsales\Eshop\Core\Registry;
+
+use Wirecard\PaymentSdk\Transaction\SepaDirectDebitTransaction;
+
+class SepaDirectDebitPaymentMethodTest extends OxidEsales\TestingLibrary\UnitTestCase
+{
+    /**
+     * @var SepaDirectDebitPaymentMethod
+     */
+    private $_oPaymentMethod;
+
+    protected function setUp()
+    {
+        parent::setUp();
+        $this->_oPaymentMethod = new SepaDirectDebitPaymentMethod();
+    }
+
+    public function testGetConfig()
+    {
+        $oConfig = $this->_oPaymentMethod->getConfig();
+        $this->assertNotNull($oConfig);
+    }
+
+    public function testGetConfigFields()
+    {
+        $aConfigFields = $this->_oPaymentMethod->getConfigFields();
+        $this->assertArrayHasKey("sepaMandateCustom", $aConfigFields);
```

Comment:
the new convention is to only have one assert per test, you can do an array assertion

Reply:
please refactor that through the whole file

Reply:
Done!